build.assets/genproto.sh

```diff
@@ -26,6 +27,14 @@ main() {
     --path=proto/teleport/lib/multiplexer/
   buf generate --template=lib/prehog/buf.gen.yaml lib/prehog/proto
 
+  # Generate lib/teleterm & JS protos.
+  # TODO(ravicious): Refactor generating JS protos to follow the approach from above, that is have a
+  # separate call to generate Go protos and another for JS protos instead of having
+  # teleterm-specific buf.gen.yaml files.
+  # https://github.com/gravitational/teleport/pull/19774#discussion_r1061524458
+	buf generate --template=lib/prehog/buf-teleterm.gen.yaml lib/prehog/proto
+	buf generate --template=lib/teleterm/buf.gen.yaml lib/teleterm/api/proto
```

Comment:
Originally I wanted to do this in this PR. But the list of changes is already big enough so I'll handle that in a separate PR (which I'm working on at the moment).